# NOTE: All the variables here must have the same names as in CameraTypes.hpp:CameraState
# When setting some paramter to the camera, we need to specify which one we want
# to set

float32 gain
float32 exposureUs
int16 frameRate
float32 redGain
float32 greenGain
float32 blueGain
bool autoGainEnabled
bool autoExposureEnabled
bool autoWhiteBalanceEnabled